README.md

```diff
@@ -3,22 +3,22 @@
 **Kover** - Gradle plugin for Kotlin code coverage agents: [IntelliJ](https://github.com/JetBrains/intellij-coverage)
 and [JaCoCo](https://github.com/jacoco/jacoco).
 
-Minimal supported Gradle version: `6.4`. 
+Minimal supported `Gradle` version: `6.4`. 
 
 ## Table of content
 - [Features](#features)
 - [Quickstart](#quickstart)
-    - [Apply plugin to single-module project](#apply-plugin-to-single-module-project)
-        - [Applying plugins with the plugins DSL](#applying-plugins-with-the-plugins-dsl)
-        - [Legacy Plugin Application: applying plugins with the buildscript block](#legacy-plugin-application-applying-plugins-with-the-buildscript-block)
-    - [Apply plugin to multi-module project](#apply-plugin-to-multi-module-project)
+  - [Apply plugin to single-module project](#apply-plugin-to-single-module-project)
+    - [Applying plugins with the plugins DSL](#applying-plugins-with-the-plugins-dsl)
+    - [Legacy Plugin Application: applying plugins with the buildscript block](#legacy-plugin-application-applying-plugins-with-the-buildscript-block)
+  - [Apply plugin to multi-module project](#apply-plugin-to-multi-module-project)
 - [Configuration](#configuration)
-  - [JVM test task](#configuring-jvm-test-task)
-  - [Reports](#configuring-reports)
-  - [Reports collecting](#configuring-reports-collecting)
-  - [Entire plugin](#configuring-entire-plugin)
-- [Verification](#verification)
-- [Tasks](#tasks)
+  - [Configuring JVM test task](#configuring-jvm-test-task)
+  - [Configuring reports](#configuring-reports)
+  - [Configuring reports collecting](#configuring-reports-collecting)
+  - [Configuring entire plugin](#configuring-entire-plugin)
+    - [Verification](#verification)
```

Comment:
Verification and the list of tasks do not relate to configuring the general behavior of the plugin (settings for all modules), so it is better to leave them at the root level.

Reply:
Done